lib/wallet/http.js

```diff
@@ -1427,6 +1427,99 @@ class HTTP extends Server {
         errorMessages: errorMessages});
     });
 
+    // Create Batch Finish (Redeem or Register)
+    this.post('/wallet/:id/batch/finish', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      let names = valid.array('names');
```

Comment:
names seems misleading to me because it's not the same as the `names` variable in batch opens/reveals. Maybe `const finishRequests = valid.array('finishRequests')`?